core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationTutorialFirstTimeService.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for the state Translation tutorial.
+ */
+
+oppia.factory('StateTranslationTutorialFirstTimeService', [
```

Comment:
This entirely looks similar to `StateEditorTutorialFirstTimeService`, can we make a single service which can handle all the tutorial?  (It's okay if there's any issue doing this, but would love to know!)

Reply:
Yeah we can do that.

Reply:
Sounds good! :)